pnpm-workspace.yaml

```diff
@@ -1,2 +1,8 @@
 packages:
-  - './packages/*'
\ No newline at end of file
+  - './packages/*'
+
+autoInstallPeers: false
```

Comment:
This option reverts to the value that used to be the default in pnpm 7. We don't technically need it, but in my view the new default is a downgrade.